### ISSUE TITLE
[Bug] Notion importer failed - now awaiting completion of failed jobs

### ISSUE THREAD

**Issue by BeeSee20Eye35** [AppFlowy-IO/AppFlowy#6753]
### Bug Description

I exported a database page from Notion to a zip, renamed it and added it to the notion importer web page. Nothing seemed to happen for some seconds and then a message briefly flashed up. I am sorry but I failed to capture it before it disappeared. It said something like, "Failed ...." perhaps something about a request but I am not certain. 

I tried it again with the same result. (I know. I should have got the error message that time. Doh! Sorry again.)

I thought perhaps renaming the zip file had been the problem so I exported another page and this time transferred it to the importer web page without changing anything. This time, it said that there were 2 existing jobs and please wait until they are completed. Since they are failed jobs, I do not expect they will complete. 

The same happened when I tried it again.

### How to Reproduce

I suggest repeating my attempt to use the importer on the attached Notion export file I tried. 

[notion inbox.zip](https://github.com/user-attachments/files/17686415/notion.inbox.zip)

### Expected Behavior

I expected to get a converted output I could import into AppFlowy

### Operating System

Windows 11

### AppFlowy Version(s)

Using https://appflowy.io/compare/notion-vs-appflowy

**Comment by appflowy** [comment]
Hi @BeeSee20Eye35 , I can import your zip file.  Could you please try again and let me know if there are any issues.
<img width="1566" alt="image" src="https://github.com/user-attachments/assets/115ee08e-03a5-4128-a210-63dbf0b4d9ae">

**Comment by BeeSee20Eye35** [comment]
Hi

Yeah. I am still getting the same issue. I tried with a new file.
Screenshot attached. The 3 tasks mentioned are the 2 from last time several
weeks ago and today's.

This is the url I am using to get to the importer.
https://appflowy.com/import?action=import&source=notion&redirectToImport=https%3A%2F%2Fappflowy.io%2Fcompare%2Fnotion-vs-appflowy%3Fimporting%3Dtrue


Thanks for the help.
Vinton

**Comment by BeeSee20Eye35** [comment]
I am still getting this issue. I have tried it in Chrome and Firefox. The behaviour is slight different. In chrome it takes me directly to a page asking for my file to be uploaded. In Firefox it first emailed me a link to login to Appflowy - using this url took me to the upload page. Uploading my notion export zip file resulted in a message telling me 3 tasks are pending and to wait until they are completed. 

Please see screenshot of error message. 

![Image](https://github.com/user-attachments/assets/9469838c-8594-4d1b-abdb-8b6b7188dc93)

**Comment by clown-owl** [comment]
Hi,
I also encountered the same problem.

**Comment by houchs** [comment]
Yes,i'm too!

**Comment by jannisheinzmann** [comment]
Yep, same issue here. Worth noting that Notion does not give me a zip file.
